var/spack/repos/builtin/packages/rapidjson/package.py

```diff
@@ -12,14 +12,17 @@ class Rapidjson(CMakePackage):
     homepage = "https://rapidjson.org"
     url      = "https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz"
 
+    version('00dbcf2',
+            git = 'https://github.com/Tencent/rapidjson.git',
+            commit = '00dbcf2')
```

Comment:
```suggestion
    version('1.2.0-2021-08-13', git = 'https://github.com/Tencent/rapidjson.git', commit = '00dbcf2c6e03c47d6c399338b6de060c71356464')
```

This is a bit improvised... but at least this way the versions are properly ordered and more releases between 2016 and now can be added...